src/v2/api/index.md

```diff
@@ -1739,99 +1739,99 @@ type: api
+  Si la instancia de Vue no recibió la opción `el` durante la instanciación, ella estará en el estado "no montada" sin un elemento del DOM asociado. `vm.$mount()` puede ser utilizado para comenzar a montar una instancia de Vue no montada.
 
-  If `elementOrSelector` argument is not provided, the template will be rendered as an off-document element, and you will have to use native DOM API to insert it into the document yourself.
+  Si el argumento `elementOrSelector` no es provisto, la plantilla será renderizara como un elemento "fuera del documento", y ustede deberá utilizar la API nativa del DOM para insertarla en el documento.
 
-  The method returns the instance itself so you can chain other instance methods after it.
+  El método retorna la propia instancia para que usted pueda encadenar otros métodos de instacia luego.
 
-- **Example:**
+- **Ejemplo:**
 
   ``` js
   var MyComponent = Vue.extend({
     template: '<div>Hello!</div>'
   })
 
-  // create and mount to #app (will replace #app)
+  // crea y monta en #app (reemplazará #app)
   new MyComponent().$mount('#app')
 
-  // the above is the same as:
+  // el ejemplo de arriba es equivalente a:
   new MyComponent({ el: '#app' })
 
-  // or, render off-document and append afterwards:
+  // o, renderizar "fuera del documento" y agregarla luego:
   var component = new MyComponent().$mount()
   document.getElementById('app').appendChild(component.$el)
   ```
 
-- **See also:**
-  - [Lifecycle Diagram](../guide/instance.html#Lifecycle-Diagram)
-  - [Server-Side Rendering](../guide/ssr.html)
+- **Vea también:**
+  - [Diagrama de Ciclo de Vida](../guide/instance.html#Diagrama-del-Ciclo-de-vida)
+  - [Renderizado del lado de Servidor](../guide/ssr.html)
 
 ### vm.$forceUpdate()
 
 - **Uso:**
 
-  Force the Vue instance to re-render. Note it does not affect all child components, only the instance itself and child components with inserted slot content.
+  Fuerza a la instancia de Vue a renderizarse nuevamente. Note que esto no afecta a todos sus componentes hijo, solamente a la propia instancia y componentes hijos con contenido insertado en _slot_.
 
 ### vm.$nextTick( [callback] )
 
-- **Arguments:**
+- **Argumentos:**
   - `{Function} [callback]`
 
 - **Uso:**
 
-  Defer the callback to be executed after the next DOM update cycle. Use it immediately after you've changed some data to wait for the DOM update. This is the same as the global `Vue.nextTick`, except that the callback's `this` context is automatically bound to the instance calling this method.
+  Pospone la ejecución de la función _callback_ para luego del próximo ciclo de actualización del DOM. Utilize este método inmediatamente luego de modificar datos para esperar por la actualización del DOM. Este método es el mismo que el método global `Vue.nextTick`, excpeto que el contexto `this` de la función _callback_ es automáticamente vinculado a la instancia que invoco este método.
```

Comment:
There is a misspelling here:

```suggestion
  Pospone la ejecución de la función _callback_ para luego del próximo ciclo de actualización del DOM. Utilice este método inmediatamente luego de modificar datos para esperar por la actualización del DOM. Este método es el mismo que el método global `Vue.nextTick`, excepto que el contexto `this` de la función _callback_ es automáticamente vinculado a la instancia que invoco este método.
```

Reply:
Yeah, I always that mistake 🤦‍♂ I know it's wrong, but I still manage to spell it wrong 😅